migrations/20161205161306_add_auto_approval_config.js

```diff
@@ -0,0 +1,22 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.createTable('AutoApprovalConfig', function (table) {
+    table.increments('AutoApprovalConfigId')
+    table.string('Caseworker', 100)
+    table.dateTime('DateCreated')
+    table.boolean('AutoApprovalEnabled')
+    table.decimal('CostVariancePercentage')
+    table.decimal('MaxClaimTotal')
+    table.integer('MaxDaysAfterAPVUVisit')
+    table.integer('MaxNumberOfClaimsPerYear')
+    table.string('RulesDisabled', 200)
+    table.boolean('IsEnabled')
+  })
```

Comment:
after creating table add default config based on config defaults so we don't have to manually use the internal screen on all envs